### ISSUE TITLE
Occasional crash clicking on StatusBar icon

### ISSUE THREAD

**Issue by cwillisf** [trueinteractions/tint2#183]
I've been seeing an occasional crash that happens when I click on the status bar icon for my Tint2-based application. The chances seem to go up if I leave my application running for a while.

Here's what I see when it happens:
```
Cannot read property 'fireEvent' of undefined
TypeError: Cannot read property 'fireEvent' of undefined
    at StatusBar.js:27:22
    at wrap (Utilities_base.js:206:21)
    at core.js:281:31
    at callback.js:68:25

Process finished with exit code 1
```

Here's how I have it set up initially:
```js
var MyApplication = function () {
    var self = this;

    var os = require('os');

    var sb = new StatusBar();
    sb.enabled = true;
    sb.tooltip = 'MyApplication';
    sb.image = 'app://my-application.png';
    sb.addEventListener('click', function () { self.showUI(); });
    application.exitAfterWindowsClose = false;
```

Then I initialize a PopOver with a WebView in it, which is what I pop up with that `self.showUI()` call.

Honestly I'm hoping that this is my fault, but I'm not sure what I could be doing which could cause `callback` to become `undefined` here:

https://github.com/trueinteractions/tint2/blob/master/modules/StatusBar/StatusBar_mac.js#L27
```js
    tintStatusBarDelegate.addMethod('click:','v@:@', 
      utilities.errorwrap(function(self,_cmd,frame) { 
        self.callback.fireEvent('click');
    }));
```

Any help at all would be appreciated. Thank you!

**Comment by trevorlinton** [comment]
@cwillisf We've been seeing this as well, it appears the objective-c block is re-collected or V8 recollects the delegate callback, were looking into this to see if we can find a potential permanent solution.

Hopefully we'll have this fixed in the next release (1/25/2016)

**Comment by trevorlinton** [comment]
This is fixed in v2.3.2 you can dowload in the releases section.  Feel free to reopen if you have any further issues.
